<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreDiscoverItem</class>
 <widget class="QWidget" name="StoreDiscoverItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>76</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>76</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(237, 237, 237);</string>
  </property>
  <layout class="QHBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="coverLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>136</width>
       <height>76</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>136</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::PlainText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="infoFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="infoFrameLayout" columnstretch="1,0,0,0">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <item row="1" column="0" rowspan="2" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignBottom">
       <widget class="QWidget" name="statusHolder" native="true">
        <property name="styleSheet">
         <string notr="true">QLabel
{
	color: white;
	padding-bottom: 7px;
	padding-left: 16px;
	padding-right: 16px;
	padding-top: 7px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="statusLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
       <widget class="QLabel" name="upcomingLabel">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(9, 74, 119);</string>
        </property>
        <property name="text">
         <string>SOON</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
         </item>
         <item>
          <widget class="QLabel" name="wishlistedLabel">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #eca125;</string>
           </property>
           <property name="text">
            <string>WISHLISTED</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ownedLabel">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #c7c7c7; color: #5c5c5c;</string>
           </property>
           <property name="text">
            <string>IN LIBRARY</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="statusSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
 </layout>
 </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="newPriceLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="oldPriceLabel">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <strikeout>true</strikeout>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="2" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
       <widget class="QPushButton" name="addToCartButton">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(180, 230, 60), stop:1 rgb(140, 202, 39));
padding: 8px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/cart-plus.svg</normaloff>:/icons/cart-plus.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2" alignment="Qt::AlignmentFlag::AlignVCenter">
       <widget class="QLabel" name="discountLabel">
        <property name="styleSheet">
         <string notr="true">color: white; 
background-color: rgb(110, 29, 114); 
border-radius: 3px;
padding: 10px;</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
